tensorflow/core/kernels/data/name_utils.cc

```diff
@@ -37,10 +37,16 @@ string OpName(const string& dataset_type) {
     return "MapDataset";
   } else if (dataset_type == "PaddedBatch") {
     return "PaddedBatchDataset";
+  } else if (dataset_type == "ParallelInterleave") {
```

Comment:
should this be `ParallelInterleaveDatasetV2`?

Reply:
My mistake! Yes, they should match with each other. Should `kDatasetType` be `ParallelInterleaveDatasetV2` or `ParallelInterleaveDataset`?